src/runtime/audience-action-flow.js

```diff
@@ -267,12 +270,45 @@ export class AudienceActionFlow {
   // eslint-disable-next-line no-unused-vars
   handleSurveyDataTransferRequest_(request) {
     // @TODO(justinchou): execute callback with setOnInterventionComplete
-    // and Google Analytics, then check for success
+    // then check for success
+    const gaLoggingSuccess = this.logSurveyDataToGoogleAnalytics(request);
     const surveyDataTransferResponse = new SurveyDataTransferResponse();
-    surveyDataTransferResponse.setSuccess(true);
+    surveyDataTransferResponse.setSuccess(gaLoggingSuccess);
     this.activityIframeView_.execute(surveyDataTransferResponse);
   }
 
+  /**
+   * Logs SurveyDataTransferRequest to Google Analytics. Returns boolean
+   * for whether or not logging was successful.
+   * @param {SurveyDataTransferRequest} request
+   * @return {boolean}
+   * @private
+   */
+  logSurveyDataToGoogleAnalytics(request) {
+    const ga = this.deps_.win().ga || null;
+    const gtag = this.deps_.win().gtag || null;
+    if (!isFunction(ga) && !isFunction(gtag)) {
+      return false;
+    }
+    request.getSurveyQuestionsList().map((question) => {
+      const answer = question.getSurveyAnswersList()[0];
+      const event = {
+        eventType: AnalyticsEvent.ACTION_SURVEY_DATA_TRANSFER,
+        eventOriginator: EventOriginator.SWG_CLIENT,
+        isFromUserAction: true,
+        additionalParameters: null,
+      };
+      const analyticsParams = {
```

Comment:
Should this be put in the `googleAnalyticsParameters` property?

Reply:
yes, good catch!